testing.json

```diff
@@ -0,0 +1,151 @@
+{ "type": "FeatureCollection",
+
+    "features": [
+      { 
+        "type": "Feature",
+        "geometry": {
+          "type": "Point", 
+          "coordinates": [-75.134729, 39.974192]
+          },
+          "properties": {
+            "name": "ArtWell",
+            "address": "100 W Oxford St",
+            "ZIP": "19122",
+            "website": "http://www.theartwell.org/",
+            "grant-amount": "$50,000"
+          }
+        },
+      { "type": "Feature",
+        "geometry": {
+          "type": "Point", 
+          "coordinates": [-75.142764, 39.953009]
+          },
+          "properties": {
+            "name": "Clay Studio",
+            "address": "139 N 2nd St",
+            "ZIP": "19106",
+            "website": "https://www.theclaystudio.org/",
+            "grant-amount": "$50,000"
```

Comment:
It's conventional to use what's known as "Camel Case" for javascript and JSON, which simply is a style that removes whitespace/dash/underscore with a capitalized letter.  Thus, this would more typically be written as `grantAmount` than `grant-amount`.  Different languages tend to have different conventions, for instance Python would prefer and underscore `grant_amount`.